opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -0,0 +1,337 @@
+					point.value.distribution_value.mean = agg_data.mean_data
+					point.value.distribution_value.sum_of_squared_deviation = agg_data.sum_of_sqd_deviations
+
+					# Uncomment this when stackdriver supports Range
+					# point.value.distribution_value.range.min = agg_data.min
+					# point.value.distribution_value.range.max = agg_data.max
+
+					point.value.distribution_value.bucket_options.explicit_buckets.bounds.extend(list(map(float,agg_data.bounds)))
+					point.value.distribution_value.bucket_counts.extend(list(map(int,agg_data.counts_per_bucket)))
+				elif type(tag_value.value) is int:
+					point.value.int64_value = int(tag_value.value)
+				elif type(tag_value.value) is float:
+					point.value.float_value = float(tag_value.value)
+				elif type(tag_value.value) is str:
+					point.value.string_value = str(tag_value.value)
+
+				start = datetime.datetime.strptime(view_data.start_time,"%Y-%m-%dT%H:%M:%S.%fZ")
+				end = datetime.datetime.strptime(view_data.end_time,"%Y-%m-%dT%H:%M:%S.%fZ")
+
+				timestamp_start = (start - datetime.datetime(1970, 1, 1)).total_seconds()
+				timestamp_end = (end - datetime.datetime(1970, 1, 1)).total_seconds()
+
+				point.interval.end_time.seconds = int(timestamp_end)
+				point.interval.end_time.nanos = int((timestamp_end - point.interval.end_time.seconds) * 10**9)
+
+				#
+				# Uncomment this when LastValue gets supported
+				#
+				#if not agg.aggregation_type is aggregation.Type.LASTVALUE:
+				if timestamp_start == timestamp_end: # avoiding start_time and end_time to be equal
+					timestamp_start = timestamp_start - 1
+				point.interval.start_time.seconds = int(timestamp_start)
+				point.interval.start_time.nanos = int((timestamp_start - point.interval.end_time.seconds) * 10**9)
+
+			time_series.append(series)
+
+			request = {}
+			request[CONS_NAME] = self.client.project_path(self.options.project_id)
+			request[CONS_TIME_SERIES] = time_series
+			requests.append(request)
+
+			if len(time_series) == int(limit):
+				time_series = []
+		return requests
+
+	def create_metric_descriptor(self, view):
+		"""
+		it creates a MetricDescriptor 
+		for the given view data in Stackdriver Monitoring.
+		An error will be raised if there is 
+		already a metric descriptor created with the same name
+		but it has a different aggregation or keys.
+		"""
+		view_measure = view.measure
+		view_aggregation = view.aggregation
+		tag_keys = view.columns
+		view_name = view.name
+
+		metric_type = namespaced_view_name(view_name)
+		value_type = None
+		unit = view_measure.unit
+		# Default metric Kind
+		metric_kind = monitoring_v3.enums.MetricDescriptor.MetricKind.CUMULATIVE
+
+		if view_aggregation.aggregation_type is aggregation.Type.COUNT:
+			value_type = monitoring_v3.enums.MetricDescriptor.ValueType.INT64
+			# If the aggregation type is count, which counts the number of recorded measurements, 
+			# the unit must be "1", because this view does not apply to the recorded values.
+			unit = 1
```

Comment:
Isn't this supposed to be the string "1" instead of the int 1? The comment above it, also seems to allude to "1"

Reply:
I think it is just a comment error, because it really should be a number.